apps/ledger-live-mobile/src/screens/Settings/index.tsx

```diff
@@ -23,31 +25,31 @@ export default function Settings({
   ScreenName.SettingsScreen
 >) {
   const { t } = useTranslation();
+  const dispatch = useDispatch();
   const accounts = useSelector(accountsSelector);
   const { handleSettingsRateApp } = useRatings();
 
-  const [debugVisible, setDebugVisible] = useState(
-    Config.FORCE_DEBUG_VISIBLE || false,
-  );
+  const debugVisible =
+    useSelector(isDebugMenuVisible) || Config.FORCE_DEBUG_VISIBLE;
   const count = useRef(0);
   const debugTimeout = useRef(onTimeout);
 
   function onTimeout(): void {
     timer.timeout(() => {
       count.current = 0;
-    }, 1000);
+    }, 2000);
   }
 
   const onDebugHiddenPress = useCallback(() => {
     if (debugTimeout) debugTimeout.current();
     count.current++;
     if (count.current > 6) {
       count.current = 0;
-      setDebugVisible(!debugVisible);
+      dispatch(openDebugMenu());
```

Comment:
before, pressing again more than 6 times a 2nd time would close the debug menu. It's not the case anymore. Is it a regression ? .... i'm joking no worries 🙈